src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import { FunctionComponent } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../react-app-env';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: FunctionComponent<Props> = ({ todo }) => {
+  const {
+    title,
+    id,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+      {user && <UserInfo user={user} />}
```

Comment:
```suggestion
      {user && (
        <UserInfo user={user} />
      )}
```

Reply:
I don`t think in this case it required to use of parentheses) BTW mentor will tell for sure 😄